.github/renovate.json5

```diff
@@ -1,33 +1,62 @@
 {
-  "extends": [
+  extends: [
     // https://docs.renovatebot.com/presets-config/#configjs-lib
-    "config:js-lib",
+    'config:js-lib',
 
     // https://docs.renovatebot.com/presets-default/#maintainlockfilesweekly
-    ":maintainLockFilesWeekly",
+    ':maintainLockFilesWeekly',
 
     // https://docs.renovatebot.com/presets-default/#preservesemverranges
-    ":preserveSemverRanges",
+    ':preserveSemverRanges',
 
     // https://docs.renovatebot.com/presets-npm/#npmunpublishsafe
-    "npm:unpublishSafe",
+    'npm:unpublishSafe',
   ],
-  "schedule": [
-    "after 10pm every weekday",
-    "before 5am every weekday",
-    "every weekend",
+  schedule: [
+    'after 10pm every weekday',
+    'before 5am every weekday',
+    'every weekend',
```

Comment:
```suggestion
    'every weekend',
```

Maybe we should remove weekends 😹

Reply:
PLEASE 😂

Reply:
<img width="372" alt="Screenshot 2025-04-30 at 9 56 01 AM" src="https://github.com/user-attachments/assets/c3bfecd0-864c-4e89-86d4-dd441d0fd93a" />

Thank you for your service 🫡